src/main/java/de/bayern/gdi/gui/Controller.java

```diff
@@ -445,53 +479,81 @@ private void extractBoundingBox() {
             bbox += envelope.getY() + ",";
             bbox += (envelope.getX() + envelope.getWidth()) + ",";
             bbox += (envelope.getY() + envelope.getHeight());
-            this.dataBean.addAttribute("bbox", bbox);
+            this.dataBean.addAttribute("bbox", bbox, "");
         } else {
             // Raise an error?
         }
     }
 
+
+    private boolean validateInput() {
+        String failed = "";
+        ArrayList<DataBean.Attribute> attributes
+                                = this.dataBean.getAttributes();
+        boolean ret = true;
```

Comment:
Remove ret variable. Write "return true;" at the end and "return false;" in the 'if (!failed.equals("")) { ... } instead.